### DISCUSSION TITLE
Favorite Must Learn KQL book version

### DISCUSSION THREAD

**Discussion by rod-trent** [rod-trent/MustLearnKQL#12] [Polls]
I'm curious which book versions you find the most useful.

**Comment by thinkdreams** [comment]
Currently paperback. I'm waiting for the hardcover to be released on Amazon for the 4.x version and then that may be my new favorite :)

**Reply by rod-trent** [reply]
The updated hardcover is available.

**Reply by thinkdreams** [reply]
Perfect! I've been doing something neat lately - I go and give blood and I use the Amazon gift cards they hand out to buy my study guides and books with it for furthering my knowledge of new stuff - win win for both me and the recipient of my blood :)

I was stagnant for a number of years in a previous job - and my new position at a new company has given me new life to start studying again.....

**Reply by rod-trent** [reply]
That is an awesome tip! 

Help yourself by helping someone else.

That's so cool.
